.github/workflows/tests.yml

```diff
@@ -55,10 +62,9 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 11.0.8
-
       - name: Run tests
         run: |
-          ./gradlew test --info --stacktrace --continue \
+          SKIP_JAVADOC=true ANDROID_SQLITE_DIR=$HOME/sqlite ./gradlew test --info --stacktrace --continue \
```

Comment:
If we add a new parameter for building command, what about update `README.md`, wiki and website too?

Reply:
I'll try to make it a git submodule and drop the param.

Reply:
Done, it's a submodule now, much cleaner :)